blockbase/assets/alignments-editor.css

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
This file can be removed after https://github.com/WordPress/gutenberg/pull/26853 is merged.